### DISCUSSION TITLE
Instagram store load and Outlook - AdGuardHome.

### DISCUSSION THREAD

**Discussion by marcosptassis** [AdguardTeam/AdGuardHome#2746] [Q&A]
I have been having problems with AdGuardHome, Instagram does not load stores and Outlook presents a bloquerio page, I have already tested changing the DNS on the machine and it immediately returns to normal. I am new and I count on the help of the Doctors.
![insta](https://user-images.githubusercontent.com/65137973/108516322-da716680-72a4-11eb-9d58-9802f0c2fecf.png)

**Comment by Aikatsui** [comment]
1. Make sure you have not blocked particular services through `Filters --> Blocked services`
2. Check query log and unblock affected domains
   1. Go to 'Query Log' and refresh the page
   2. Open instagram and outlook websites
   3. Back to query log and refresh the page
   4. Check affected domains and unblock
   
 Let us know if that doesn't help to solve your problem
